packages/ngtools/webpack/package.json

```diff
@@ -31,10 +31,13 @@
     "webpack-sources": "1.2.0"
   },
   "peerDependencies": {
+    "@angular/compiler-cli": ">= 5.0.0 <8.0.0",
```

Comment:
I think this will cause unmet peerdep errors in CLI projects, because devkit/build-angular (who imports ngtools/webpack) does not install angular/compiler-cli. Can you check this with a local build please?

Reply:
It doesn't like beta versions at the moment (fixing that), but otherwise no peer dependency warnings.